test/sidecar_test.go

```diff
@@ -64,12 +64,12 @@ func TestSidecarTaskSupport(t *testing.T) {
 				tb.TaskSpec(
 					tb.Step(
 						primaryContainerName,
-						"ubuntu",
+						"busybox:1.31.0-musl",
```

Comment:
Just curious, why was the container image changed?

Reply:
Found it: https://github.com/tektoncd/pipeline/issues/1253#issuecomment-527464092